OneCommunityGlobal/HighestGoodNetworkApp
# Kaikanes lessonform design

# Description
This is the design of the Lesson form page from Phase II. It was created based off the prototype below:
<img width="521" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114305309/66d08921-0151-4f32-863e-7e9e34ddcd82">

## Related PRS (if any):
N/A

## Main changes explained:
-Created route
-Created Lesson Page to resemble figma

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as anyone/everyone
4. go to http://localhost:3000/bmdashboard/lessonform
5. check the design against the prototype, check for responsiveness in different screen sizes
6. make sure the new line `overflow: hidden;` in App.css doesn't negatively affect any other pages
## Screenshots or videos of changes:
<img width="808" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114305309/4a1f6624-2751-4897-a775-cc83c61513a0">

## Note:
No functionality as of now
Working on Drag and drop which  will be a separate PR
